packages/ui/test/services/userPermissions.test.ts

```diff
@@ -1,7 +1,7 @@
 import { expect } from "@jest/globals"
```

Comment:
Should this test file be next to the new location of `packages/common/utils/userPermissions.ts` `packages/common/utils/userPermissions.test.ts`